src/commands/addons/config.js

```diff
@@ -188,8 +188,8 @@ async function update({ addonName, currentConfig, newConfig, settings, accessTok
   try {
     // TODO update updateAddon to https://open-api.netlify.com/#operation/updateServiceInstance
     updateAddonResponse = await updateAddon(settings, accessToken)
-  } catch (e) {
-    error(e.message)
+  } catch (error_) {
```

Comment:
Another autofix that changes lots of files in this PR.
This enforces consistent names for `error`. When there are several `error` in scope, it appends an `_`. Not a big fan of that style, but at least, it's consistent.
See [rule documentation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/catch-error-name.md).